iot-hub/Samples/service/DigitalTwinClientSamples/TemperatureController/TemperatureController.csproj

```diff
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="CommandLineParser" Version="2.8.0" />
+    <PackageReference Include="Microsoft.Azure.Devices" Version="1.22.1-pnp-preview" />
```

Comment:
just fyi - private nuget versions here: https://github.com/Azure/azure-iot-sdk-csharp/pull/1576

Reply:
updated to 1.23.0-pnp-preview